tests/js/setup.ts

```diff
@@ -69,7 +69,6 @@ jest
   .spyOn(performanceForSentry, 'VisuallyCompleteWithData')
   .mockImplementation(props => props.children as ReactElement);
 jest.mock('scroll-to-element', () => jest.fn());
-jest.mock('sentry/utils/search/searchBoxTextArea');
```

Comment:
i don't think this is being used anymore?

Reply:
yes it seems it is no longer used.